rectools/model_selection/cross_validate.py

```diff
@@ -61,6 +61,8 @@ def cross_validate(  # pylint: disable=too-many-locals
     filter_viewed: bool,
     items_to_recommend: tp.Optional[ExternalIds] = None,
     prefer_warm_inference_over_cold: bool = True,
+    ref_models: tp.Optional[tp.List[str]] = None,
```

Comment:
We'll need a "happy path" test of these parameters in `tests.model_selection.test_cross_validate`
Something like `test_happy_path_with_intersection`.